war/src/main/webapp/scripts/yui/container/container-debug.js

```diff
@@ -4895,7 +4895,7 @@ version: 2.9.0
     * @final
     * @type String
     */
-    Tooltip.CSS_TOOLTIP = "yui-tt";
+    Tooltip.CSS_TOOLTIP = "jenkins-tooltip";
```

Comment:
Will this break plugin compatibility?

Reply:
Also, if this is used mainly for JS I'd put a name like `js-jenkins-tooltip`

Reply:
Seems 1 plugin is using it https://github.com/search?q=org%3Ajenkinsci+yui-tt+-repo%3Ajenkinsci%2Fjenkins+-path%3Atarget+-repo%3Ajenkinsci%2Fhtmlunit&type=code

Reply:
I wouldn't worry about that plugin